Frontend/src/common/utils.js

```diff
@@ -253,21 +253,16 @@ const removeDuplicateEvents = (events) => {
     const uniqueLists = [];
     const seenLists = new Set();
 
-    for (const sublist of events) {
-        const sublistString = JSON.stringify(sublist);
+    for (let i = 0; i < events.length; i++) {
+        let obj = events[i];
+        const sublistString = JSON.stringify(obj["sync"]);
         if (!seenLists.has(sublistString)) {
```

Comment:
Say here we had an "else" block, where we would add all the async events from obj to the corresponding object in uniqueLists, would this allow us to have all the aysnc sections showing properly (i.e. ECOR 4995 A, B, etc.)?

Reply:
Ya this could work, I can work on this later